src/main/java/internetShop/BucketHandler.java

```diff
@@ -0,0 +1,33 @@
+package internetShop;
+
+import java.util.Scanner;
+
+public class BucketHandler {
+    private static Bucket bucket = new Bucket();
+
+    private static Scanner sc = new Scanner(System.in);
+
+    public static void addToTheBucket(int num) {
+        Product product = ProductListHandler.getProd().get(num);
+        if (product != null) {
```

Comment:
!Objects.isNull(product)

Reply:
or a more interesting  approach, wrap get(num) to Optional
then get(num).ifPresent(Bucket::addProduct)